tests/tamr_client/fake_json/dataset/test_dataset/test_create_dataset_already_exists.json

```
@@ -0,0 +1,22 @@
+[
+    {
+        "request": {
+            "method": "POST",
+            "path": "datasets",
+            "json": {
+                "name": "new dataset",
+                "keyAttributeNames": [
+                    "primary_key"
+                ],
+                "description": "a new dataset",
+                "externalId": null
+            }
+        },
+        "response": {
+            "status": 400,
+            "json": {
+                "message": "Dataset \"tc test dataset\" already exists"
```

nit: should this be:

```suggestion
                "message": "Dataset \"new dataset\" already exists"
```

---

Will change, just a minute.